org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporter.java

```diff
@@ -433,7 +440,9 @@ public static BuildConfiguration getBuildConfiguration(Path rootFolder) {
 		Preferences preferences = getPreferences();
 		File javaHome = getJavaHome(preferences);
 		File gradleUserHome = getGradleUserHomeFile();
-		List<String> gradleArguments = preferences.getGradleArguments();
+		List<String> gradleArguments = new ArrayList<>();
+		gradleArguments.addAll(getProtobufInitScriptArgs());
```

Comment:
We should check if the gradle arguments already include --initScript.

Reply:
It should be fine to have multiple `--init-script`, as described in Gradle doc: https://docs.gradle.org/current/userguide/init_scripts.html#sec:using_an_init_script

> The command line option can appear more than once, each time adding another init script.

Reply:
Good.